src/pgrn-list-broken-indexes.c

```diff
@@ -93,10 +94,83 @@ PGrnIsBrokenLexicon(Relation index)
 	return false;
 }
 
+static bool
+PGrnIsBrokenJsonbResource(Relation index, unsigned int nthAttribute)
+{
+	if (PGrnIsBrokenTable(
+			PGrnJSONBLookupValuesTable(index, nthAttribute, ERROR)))
+	{
+		return true;
+	}
+	if (PGrnIsBrokenTable(
+			PGrnJSONBLookupPathsTable(index, nthAttribute, ERROR)))
+	{
+		return true;
+	}
+	if (PGrnIsBrokenTable(
+			PGrnJSONBLookupTypesTable(index, nthAttribute, ERROR)))
+	{
+		return true;
+	}
+
+	if (PGrnIsBrokenTable(
+			PGrnJSONBLookupFullTextSearchLexicon(index, nthAttribute, ERROR)))
+	{
+		return true;
+	}
+	if (PGrnIsBrokenTable(
+			PGrnJSONBLookupStringLexicon(index, nthAttribute, ERROR)))
+	{
+		return true;
+	}
+	if (PGrnIsBrokenTable(
+			PGrnJSONBLookupNumberLexicon(index, nthAttribute, ERROR)))
+	{
+		return true;
+	}
+	if (PGrnIsBrokenTable(
+			PGrnJSONBLookupBooleanLexicon(index, nthAttribute, ERROR)))
+	{
+		return true;
+	}
+	if (PGrnIsBrokenTable(
+			PGrnJSONBLookupSizeLexicon(index, nthAttribute, ERROR)))
+	{
+		return true;
+	}
+	return false;
+}
+
 static bool
 PGrnIndexIsBroken(Relation index)
 {
-	return PGrnIsBrokenSources(index) || PGrnIsBrokenLexicon(index);
+	TupleDesc desc;
+	unsigned int i;
+	bool isJsonb = false;
+
+	if (PGrnIsBrokenSources(index))
+	{
+		return true;
+	}
+
+	desc = RelationGetDescr(index);
+	for (i = 0; i < desc->natts; i++)
+	{
+		Form_pg_attribute attribute = TupleDescAttr(desc, i);
+		if (PGrnAttributeIsJSONB(attribute->atttypid))
```

Comment:
We can simplify `jsonb` check like:

https://github.com/pgroonga/pgroonga/blob/52e7d2ed997b2e32760a6163083d9559e89115f5/src/pgroonga.c#L4723-L4724

Because we don't support multi-column `jsonb` index.